### ISSUE TITLE
Syntax highlighting broken for regex with escaped backslash

### ISSUE THREAD

**Issue by tehrengruber** [JuliaEditorSupport/Julia-sublime#5]
Typing `r"\\\\"` inside a julia file breaks syntax highlighting for the rest of the file even though it is valid julia syntax.

![screenshot from 2016-03-09 23-15-47](https://cloud.githubusercontent.com/assets/6791445/13652542/0d0cb534-e64d-11e5-8055-ba24df0868b4.png)

**Comment by epitron** [comment]
I just updated the package, using the syntaxes from https://github.com/JuliaLang/Julia.tmbundle

Tell me if that fixes it!

**Comment by tehrengruber** [comment]
I updated the package, but the problem persists.

**Comment by epitron** [comment]
Try out a different package - Sublime-IJulia has a more complex syntax
highlighting definition than mine does.

**Comment by ViktorQvarfordt** [comment]
The above fix applies to all prefixed strings, not just `r"..."` regex strings.

As far as I can tell, all syntax files originate from Julia.tmbundle, including ours. I'll try to make the syntax highlighting definition a unified effort among these projects.

**Comment by ViktorQvarfordt** [comment]
Sorry, my fix was bad, I've reverted it. I though this was something simple. It appears non-trivial to detect if the `\"` is at the end of the regex or in the middle (where it currently works as intended).

I'll hear with the people maintaining Julia.tmbundle. I just checked their code and they have the same problem.

(The Sublime-IJulia is very old, their `Julia.tmLanguage` was last updated Dec 22, 2014.)
